core/src/impl/Kokkos_CPUDiscovery.cpp

```diff
@@ -55,29 +55,14 @@
 namespace Kokkos {
 namespace Impl {
 
-// The following function (processors_per_node) is copied from here:
-// https://lists.gnu.org/archive/html/autoconf/2002-08/msg00126.html
-// Philip Willoughby
-
 int processors_per_node() {
-#ifdef _WIN32
```

Comment:
I suppose we should drop https://github.com/kokkos/kokkos/blob/0d1f546f20f44d939e8dac34833266912147c027/core/src/impl/Kokkos_CPUDiscovery.cpp#L44-L46 as well in that case.

Reply:
We are planning to put it back in once we figured out how to do it in windows correctly.